cash/cash.c

```diff
@@ -0,0 +1,41 @@
+#include <cs50.h>
+#include <stdio.h>
+
+int main(void)
+{
+    int change;
+
+    do
+    {
+        change = get_int("Change owed: ");
+    }
+    while (change <= 0);
+
+    int quarters = 0;
+    int dimes = 0;
+    int nickels = 0;
+    int pennies = 0;
+
+    while (change >= 25)
+    {
+        quarters++;
+        change -= 25;
+    }
+    while (change >= 10)
+    {
+        dimes++;
+        change -= 10;
+    }
+    while (change >= 5)
```

Comment:
I would highly recommend to do extra lines between any conditional statements
```suggestion

        while (change >= 5)
```